From 4d953c280ce75beef331df588d85e03ec788432f Mon Sep 17 00:00:00 2001
From: Vijaya Ramarao Mellempudi <Vijaya_RamaraoMellempudi@comcast.com>
Date: Fri, 22 Feb 2019 11:45:00 +0000
Subject: [PATCH] RDK-23442:Increasing the return_buff size to 16KB :Increasing
 the return_buff size to 96KB Increase scan results buf size to 96KB.

Source: COMCAST ab23b5fcb84ebbe1697b423bee0ec7dfdfb66bad Mar 6, 2019 12:08 PM
Signed-off-by: Vijaya Ramarao Mellempudi <Vijaya_RamaraoMellempudi@comcast.com>

---
 wpa_supplicant/bss.c        | 3 +--
 wpa_supplicant/ctrl_iface.c | 2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/wpa_supplicant/bss.c b/wpa_supplicant/bss.c
index e13783c..3160c0f 100644
--- a/wpa_supplicant/bss.c
+++ b/wpa_supplicant/bss.c
@@ -455,8 +455,7 @@ static struct wpa_bss * wpa_bss_add(struct wpa_supplicant *wpa_s,
 	os_memcpy(bss->ies, res + 1, res->ie_len + res->beacon_ie_len);
 	wpa_bss_set_hessid(bss);
 
-	if (wpa_s->num_bss + 1 > wpa_s->conf->bss_max_count &&
-	    wpa_bss_remove_oldest(wpa_s) != 0) {
+	if (wpa_s->num_bss + 1 > wpa_s->conf->bss_max_count) {
 		wpa_printf(MSG_ERROR, "Increasing the MAX BSS count to %d "
 			   "because all BSSes are in use. We should normally "
 			   "not get here!", (int) wpa_s->num_bss + 1);
diff --git a/wpa_supplicant/ctrl_iface.c b/wpa_supplicant/ctrl_iface.c
index bcd67fc..219cf9c 100644
--- a/wpa_supplicant/ctrl_iface.c
+++ b/wpa_supplicant/ctrl_iface.c
@@ -11447,7 +11447,7 @@ char * wpa_supplicant_ctrl_iface_process(struct wpa_supplicant *wpa_s,
 					 char *buf, size_t *resp_len)
 {
 	char *reply;
-	const int reply_size = 4096;
+	const int reply_size = 96*1024;
 	int reply_len;
 
 	if (os_strncmp(buf, WPA_CTRL_RSP, os_strlen(WPA_CTRL_RSP)) == 0 ||
